android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/datatypes/DoorStatusTests.java

```diff
@@ -34,7 +32,7 @@ public void testRpcValues() {
 
         // Valid Tests
         assertTrue(Validator.validateGrid(msg.getLocation(), location));
```

Comment:
@kostyaBoss You can delete line 34:  `assertTrue(Validator.validateGrid(msg.getLocation(), location));`.